### PR TITLE
[NO QA] Update contributing guidelines for bug reporting

### PR DESCRIPTION
**Pull request by Julesssss**
cc @mountiny 

### Details

Simply updates our guidelines to better explain how bugs are reported. 

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/287121

### Tests

N/A

### QA Steps

N/A

### PR Author Checklist
N/A

### CODE REVIEW

File: contributingGuides/CONTRIBUTING.md [hunk 1]
```diff
@@ -181,9 +181,19 @@ Follow all the above above steps and processes. When you find a job you'd like t
...
+- **LHN:** Left Hand Navigation (Primary navigation modal in Expensify Chat, docked on the left hand side)
+- **OP:** Original Post (Most commonly the post in E/App GH issues that reports the bug)
+- **GBR:** Green Brick Road (UX Design Principle that utlizes green indicators on action items to encourage the user down the optimal path for a given process or task) 
+- **RBR:** Red Brick Road (UX Design Principle that utlizes red indicators on action items to encourage the user down the optimal path for handling and discovering errors) 
+- **VBA:** Verified Bank Account (Bank account that has been verified as real and belonging to the correct business/individual)
+- **NAB:** Not a Blocker (An issue that doesn't block progress, but would be nice to not have)
+- **LHN:** Left Hand Navigation
+- **IOU:** I owe you (used to describe payment requests between users)
+- **OTP:** One time password, or magic sign-in
+- **RDM:** Right Docked Modal (on larger screens, pages are often displayed docked to the right side of the screen)
```
Comment (mountiny) [inline contributingGuides/CONTRIBUTING.md:196]:
```suggestion
- **RDM** or **RHP** Right Docked Modal / Right Hand Panel (on larger screens, pages are often displayed docked to the right side of the screen)
```

**Review by mountiny — COMMENTED** [review]
Once change requested

File: contributingGuides/CONTRIBUTING.md (hunk 1 above)
Comment (Julesssss) [reply contributingGuides/CONTRIBUTING.md:196]: Applied

**Review by mountiny — APPROVED** [review]
Thanks for the changes

**Review by madmax330 — APPROVED** [review]

**Pull request merged**